contracts/settings/AlloSettings.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.17;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+contract AlloSettings is OwnableUpgradeable {
+
+  string public constant VERSION = "1.0.0";
+
+  uint8 public constant PERCENTAGE_PRECISION = 100;
```

Comment:
Maybe it makes sense to have it as `uint8 public constant DENOMINATOR = 10000`

One general thing:
In previous audits, auditors have pointed out that there is a convention that 100% is represented as 1e18 and 10% is represented as 1e17, 1% 1e16, etc..
(Personally, I'm not married to that.)

Reply:
great point @KurtMerbeth !! I like it, I'm happy with any of these solutions!

Reply:
I like @KurtMerbeth first suggestion more than the general convention.

Reply:
Ah alright ! That's smart